src/main/resources/application.properties

```diff
@@ -454,6 +454,9 @@ spring.activemq.user=
 # Login password of the broker.
 spring.activemq.password=
 
+# Cap the amount of activities sent per message to prevent memory issues
+activemq.max.message.concept-activities=250
```

Comment:
@markrynbeek  It would be better to change this config to authoring.traceability.message.max-concepts or something similar to this.